services/service.js

```diff
@@ -0,0 +1,12 @@
+(function (angular) {
+  angular.module("MiniECommerce")
+    .service("myService", ["$http", function ($http) {
+      this.getProducts = async function getAll() {
```

Comment:
let's try to use arrow functions as much as we can. Does this work?

```
.service("myService", ["$http", ($http) => {
      this.getProducts = async getAll() => {
```

Reply:
I could have changed the getProducts function, but not the one in the service dependencies.

.service("myService", ["$http", function ($http) {
      this.getProducts = async () => {